mmv1/products/compute/RegionNetworkEndpointGroup.yaml

```diff
@@ -168,6 +168,20 @@ properties:
     custom_expand: 'templates/terraform/custom_expand/resourceref_with_validation.go.tmpl'
     resource: 'Subnetwork'
     imports: 'selfLink'
+  - name: 'pscData'
+    type: NestedObject
+    description: |
+      This field is only used for PSC NEGs.
+
+    properties:
+      - name: 'producerPort'
+        type: String
+        ignore_read: true
```

Comment:
Given what I said in my other review comment, and from looking at [the logs for the acceptance test here](https://github.com/GoogleCloudPlatform/magic-modules/pull/11820#issuecomment-2377316553), I assume that currently the API can accept and use the producerPort value but does not return that value in any responses. If that's the case then ignore_read makes sense here, but I don't have much info to go on when reviewing.

If you could confirm the above I'm happy to LGTM the PR.

Reply:
@SarahFrench, you are correct—the API can accept the new field, but it cannot return the value yet. We are currently working on this, and it may take a few weeks to roll out.

Additionally, the public documentation is slightly behind. It will be updated once everything is fully implemented.

Reply:
In that case, do you want to go ahead with this PR being merged? Or to wait until the value is also returned?

Reply:
Thanks for checking, let us wait until it is been confirmed and tested before merge.